packages/trezor-user-env-link/src/api.ts

```diff
@@ -158,6 +158,7 @@ export class TrezorUserEnvLinkClass extends TypedEmitter<WebsocketClientEvents>
         return null;
     }
     async startBridge(version?: string) {
```

Comment:
good enough, just one nitpick -  javascript way of writing this is:

```
    async startBridge(version = '2.0.33') {
```

Reply:
Right, thanks, corrected with a constant set